components/lib/dropdown/Dropdown.vue

```diff
@@ -317,7 +317,10 @@ export default {
             this.$emit('blur', event);
         },
         onKeyDown(event) {
-            if (this.disabled) {
+            // Android keyboard event
+            const isAndroidKeyboardEvent = event.isComposing || event.keyCode === 229;
```

Comment:
You can use `DomHandler.isAndroid()` instead

Reply:
@melloware
if use `DomHandler.isAndroid()`, Android's backspace key will not work
He should mean to distinguish between normal keys and input methods like Chinese and Japanese